CONTRIBUTING.md

```diff
@@ -156,9 +155,8 @@ See `metrics/tests/e2e/basicFeatures.decorator.test.ts` as an example.
 
 To run e2e tests, you can either use:
 
-* npm task `lerna-test:e2e` (`npm run lerna-test:e2e`) in root folder
-* npm task `test:e2e` (`npm run test:e2e`) in module folder (for example: `packages/metrics`) to run the module specific ones
-* jest directly `npx jest --group=e2e` in module folder. (You can run selective tests by restricting the group to the one you want. For instance `npx jest --group=e2e/metrics/decorator`)
```

Comment:
Question:

Is there a way to pass `--group` to Jest via `npm test:e2e -w packages/metrics`?  
I use this a lot to run only a specific set of E2E tests. If there is no option, I will prefer to keep this bullet if there is no alternative.

Reply:
The command above (the one with `npx`) will still work if run in the module folder (like specified by the guide).

Generally speaking, this `npm run xxx -w packages/logger -- --group=e2e/metrics/decorator` forwards all the flags after the `--` to the original scoped command.

In this specific instance, since both `test:unit` & `test:e2e` already have the `--group` flag applied (see image below):
![image](https://user-images.githubusercontent.com/7353869/200363662-5387e679-7b2d-4e15-bb18-aecd9f82404d.png)

this won't work.

We have three options here:
- Restore the line above (requires to be run within the module folder)
- Add a generic `jest` commad/script to each `pacakges/*/package.json` so that you can pass arbitrary flags and do something like this:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/7353869/200363801-fb6c7242-3692-4891-a255-5eac22847df0.png">
- Create a more scoped command/script in each `pacakges/*/package.json` called `test:group` so that you can do the same as above.

Regardless of which one we choose I'm happy to add the line back to the document.

Reply:
I'm inclined to use a generic jest command. But let's do that in another PR. 
Let's keep the change as it is.